ops/4-credhub-importer.yml

```diff
@@ -49,6 +49,13 @@
         - name: concourse/main/bosh-client
           type: value
           value: admin
+        - name: concourse/main/bosh-cpi
+          type: value
+          value: ((cpi))
+        - name: concourse/main/bosh-stemcell
+          type: value
+          value: ((stemcell))
+
```

Comment:
also add `bosh_cpi` and `bosh_stemcell`

Reply:
do we want doubles or just replacements?